lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -103,8 +103,19 @@ export class BrowserCacheManager extends CacheManager {
      * Parses passed value as JSON object, JSON.parse() will throw an error.
      * @param input
      */
-    private IsJSON(jsonValue: string): void {
-        JSON.parse(jsonValue);
+    private validateAndParseJson(jsonValue: string): object {
```

Comment:
should this be in `common` somewhere? do other libraries also want access to something like this?

Reply:
I think it's somewhat specific to the way the browser handles cache, so I don't think we need to bring it up to the common lib. If we think it is necessary in the future happy to revisit at that time.

Reply:
sounds good then!